internal/api/v1/deploy.go

```diff
@@ -315,6 +324,7 @@ func newAppIngress(appRef models.AppRef, route string) (*networkingv1.Ingress, e
 				"app.kubernetes.io/component":  "application",
 				"app.kubernetes.io/managed-by": "epinio",
 				"app.kubernetes.io/name":       appRef.Name,
+				"app.kubernetes.io/username":   username,
```

Comment:
Looking at https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/ I feel that `created-by` is the proper common label.

Reply:
Done.